internal/controlplane/identity_events.go

```diff
@@ -76,7 +76,7 @@ func HandleEvents(
 	cfg *serverconfig.Config,
 	projectDeleter projects.ProjectDeleter,
 ) {
-	d := time.Now().Add(time.Duration(10) * time.Minute)
+	d := time.Now().Add(time.Duration(5) * time.Minute)
```

Comment:
**question (non-blocking):** this is unrelated, right?

Reply:
It is unrelated to the interval, but I figured if we're querying every 5 minutes, then we want to cancel any old event processing if it's taken longer than 5 minutes, to avoid attempting the same deletions twice.

Reply:
There was another timeout on line 149; I'll send a PR for it.